src/main/java/seedu/duke/manager/StaffManager.java

```diff
@@ -55,17 +58,20 @@ public void addStaff(int staffId, String staffName, String position, double sala
      */
     public Staff findByStaffId(int staffId, boolean printMsg) {
         if (staffId <= 0) {
+            MainLogger.logInfo("Invalid input for ID.");
```

Comment:
better to use `logError` here? Also okay if we don't

Reply:
I think there's not suitable level to assign it to if we used a new `logError` (e.g. Warning might be too strong)